code/game/objects/items/plushes.dm

```diff
@@ -573,22 +573,12 @@
 		ram()
 
 /obj/item/toy/plush/goatplushie/angry/proc/ram()
-	if(prob((obj_flags & EMAGGED) ? 98:90) && isturf(loc) && considered_alive(target.mind) && !faction_check(list("goat"), target.faction, FALSE))
```

Comment:
dont remove this, only replace prob((obj_flags & EMAGGED) ? 98:90) with prob(90)

Reply:
^

Reply:
understood

Reply:
I fixed it